opentelemetry-sdk/src/opentelemetry/sdk/resources/__init__.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2019, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Any, Dict, Union
+from opentelemetry.resources import Resource
```

Comment:
I might be missing something here but is this import line correct? I'm getting an import error.

Reply:
in order for this to work, you must install the opentelemetry api package. For a reason which is not clear to me, API packages go in opentelemetry rather than opentelemetry.api.

Reply:
There's some discussion about `opentelemetry.api.trace` vs `opentelemetry.trace` in https://github.com/open-telemetry/opentelemetry-python/issues/15#issuecomment-503376530 and https://github.com/open-telemetry/opentelemetry-python/pull/55#issuecomment-512041317, the tl;dr is we generally expect people to import the API only.

We should probably add a contributing/developing doc to list gotchas like that you have to install the packages to get this to work.

Reply:
yeah, that would be great! the gotchas are fresh in my mind as well, I make try a PR